### PR TITLE
Added the new GDG logo png file for favicon

### PR DESCRIPTION
**Pull request by umeklinks**
both colored `gdg_logo.png` and gray `favicon.png`  version

The colorful version can be used if the you need a full color favicon or can use the gray version if you need a one color favicon

### CODE REVIEW

**Comment by cortinico** [comment]
Just a quick heads up: you probably need to update also those other assets:

* Different size images in [/images/manifest](/images/manifest)
* [/images/organizer-logo.svg](/images/organizer-logo.svg)
* [/images/logos/gdg-lviv.svg](/images/logos/gdg-lviv.svg)

**Comment by umeklinks** [comment]
Okay, I'll do just that. Thanks for the head up

Cheers

**Comment by ozasadnyy** [comment]
Should be *GDG Lviv* not *GDG Liv*

File: images/organizer-logo.svg [hunk 1]
```diff
@@ -1 +1,655 @@
...
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (abraham) [inline images/organizer-logo.svg]: This should render `<> GDG Lviv`

File: images/logos/gdg-liv.svg [hunk 2]
```diff
@@ -0,0 +1,453 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (abraham) [inline images/logos/gdg-liv.svg]: Updated `gdg-lviv.svg` and remove this file.

File: index.html [hunk 3]
```diff
@@ -15,7 +15,7 @@
 
   <link href="{$ url $}" rel="canonical">
 
-  <link href="images/favicon.ico" rel="icon">
```
[the remark this answers is not in the document]
Comment (umeklinks) [reply index.html]: Done

**Comment by cortinico** [comment]
Why this was closed?

**Pull request closed without merging**